src/avfoundation.cs

```diff
@@ -1078,14 +1318,17 @@ interface AVAudioOutputNode {
 
 	}	
 
-	[NoTV]
-	[NoWatch]
-	[iOS (8,0)][Mac (10,10)]
-	[BaseType (typeof (AVAudioIONode))]
-	[DisableDefaultCtor] // returned Handle is nil
-	// note: sample source (header) suggest it comes from AVAudioEngine properties
-	interface AVAudioInputNode : AVAudioMixing {
+	[Watch (4,0), TV (11,0), Mac (10,10), iOS (8,0)]
+	unsafe delegate IntPtr AVAudioIONodeInputBlock (uint frameCount);
```

Comment:
no need for `unsafe`
Let's try to figure out if `IntPtr` is the right thing to expose